packages/natds-icons/README.md

```diff
@@ -1,63 +1,76 @@
 # Natura Design System - Icons
 
-## Installation
+- How to Use
+    - For web projects
+      - Installation
+      - Using React
+      - Using as icon font
+      - Using SVG
+      - Where to find icon names
```

Comment:
acho que `where to find icon names` ta fora do react no texto mas aqui no indice ta dentro do react ;)

Reply:
Tudo errado esse indice, eu fiz ele e depois mudei tudo e esqueci de atualizar! inclusive ta sem os links tudo

Reply:
tudo errado esse índice! eu fiz depois mudei tudo e esqueci de arrumar (e de colcoar os links!)